Tasks/AzureCLIV2/task.loc.json

```diff
@@ -43,7 +43,7 @@
       "required": true,
       "helpMarkDown": "ms-resource:loc.input.help.connectedServiceNameARM",
       "properties": {
-        "EndpointFilterRule": "ScopeLevel != ManagementGroup"
+        "EndpointFilterRule": ""
```

Comment:
One more thing - I hope L104 in task will not throw if scope is management group or MLworkspace
```
var subscriptionID: string = tl.getEndpointDataParameter(connectedService, "SubscriptionID", true);
```

Reply:
nope.